src/client/components/AreaSelector/CountryList/countryList.scss

```diff
@@ -65,6 +69,10 @@
   }
 }
 
+.country-selection-list__published-date {
```

Comment:
The grid template columns above can be 1fr auto , and this maybe can be removed ?